assets/scripts_ssh_config

```diff
@@ -1,3 +1,4 @@
 StrictHostKeyChecking no
 UserKnownHostsFile /dev/null
+PasswordAuthentication no
```

Comment:
You're right. When the config drive fails it causes the entire suite to never finish. I can see the point that this is an unrelated issue, but until the test runs with a kernel that includes iso8859-1 support, it will fail. I haven't have a chance to figure out how to do that.